UPGRADE-2.0.md

```diff
@@ -176,6 +176,21 @@
       `Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository` to
       `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAssociationRepository`
 
+# Password Encoder & Salt
+The encoder and salt has been removed from the User entities. It will use the password hasher configured on Symfony security configuration.
+
+Check your user hashed passwords in your production database.
+In modern Symfony projects, the hasher name is stored on the password.
+
+Example:
+`$argon2i$v=19$m=65536,t=4,p=1$VVJuMnpUUWhRY1daN1ppMA$2Tx6l3I+OUx+PUPn+vZz1jI3Z6l6IHh2kpG0NdpmYWE`
+
+If some of your users do not have the hasher name stored in the password field you may need to configure the "migrate_from" option into Symfony, following that documentation:
```

Comment:
This needs to happen before 2.0 migrations are executed, so it needs to be somehow highlighted as super important step otherwise end-users might end with broken user tables.

Reply:
no, even if the encoder is droped from the table, the migration will be done by the Symfony configuration on the fly during the user authentication.